code/game/area/areas.dm

```diff
@@ -46,6 +46,8 @@
 	/// This gets overridden to 1 for space in area/Initialize(mapload).
 	var/always_unpowered = FALSE
 
+	var/obj/machinery/power/apc/apc = null
```

Comment:
Where is this actually assigned to?

Reply:
Hm?

Its called about 19 times.

The purpose is to hold a reference to the local apc. If there is no local apc, the original value is never changed, and stays null.